src/hotspot/os/linux/os_linux.cpp

```diff
@@ -4247,7 +4249,7 @@ jlong os::Linux::fast_thread_cpu_time(clockid_t clockid) {
 // the number of bytes written to out_fd is returned if transfer was successful
 // otherwise, returns -1 that implies an error
 jlong os::Linux::sendfile(int out_fd, int in_fd, jlong* offset, jlong count) {
-  return ::sendfile64(out_fd, in_fd, (off64_t*)offset, (size_t)count);
+  return ::sendfile64(out_fd, in_fd, (off_t*)offset, (size_t)count);
```

Comment:
Why is this continuing to use sendfile64, rather than sendfile?

Reply:
oh, great spot! I'll take you up on the recommendation for banning the *64 functions in one of the followups.